sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosItemIdentity.java

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.models;
+
+/**
+ * Encapsulates the PartitionKey/id tuple that uniquely identifies a CosmosItem
+ */
+public class CosmosItemIdentity {
```

Comment:
@kushagraThapar this is a new public api 
FYI

Reply:
We should mark this as `final` class.

Reply:
Fixed